### PR TITLE
Update nextjs lexical example

### PR DESCRIPTION
**Pull request by SugarDarius**
Updating the `nextjs-lexical` example to use the latest version of Liveblocks packages as it uses in `<Editor />` the two new components `<AnchoredThreads />` and `<FloatingThreads />` introduced in `2.3`. 

Plus updating the documentation to mark the two components available since version `2.3`.

Thanks 🙏🏻

### CODE REVIEW

File: docs/pages/api-reference/liveblocks-react-lexical.mdx [hunk 1]
```diff
@@ -258,6 +258,11 @@ function Toolbar() {
 
 ### FloatingThreads
 
+<Banner
+  title="Floating threads are available since version 2.3"
+  type="warning"
+/>
+
```
Comment (CTNicholas) [inline docs/pages/api-reference/liveblocks-react-lexical.mdx:265]:
```suggestion
```
Hey, we don't normally add banners for new features, as they'd be all over our documentation if we did! For example, we added about 25 new methods in 2.4.0.

File: docs/pages/api-reference/liveblocks-react-lexical.mdx [hunk 2]
```diff
@@ -493,6 +498,11 @@ import { Comment } from "@liveblocks/react-ui/primitives";
 
 ### AnchoredThreads
 
+<Banner
+  title="Anchored threads are available since version 2.3"
+  type="warning"
+/>
+
```
Comment (CTNicholas) [inline docs/pages/api-reference/liveblocks-react-lexical.mdx:505]:
```suggestion
```
See above!

**Review by CTNicholas — COMMENTED** [review]
Thanks for updating the example!

**Review by CTNicholas — APPROVED** [review]

File: docs/pages/api-reference/liveblocks-react-lexical.mdx (hunk 1 above)
Comment (SugarDarius) [reply docs/pages/api-reference/liveblocks-react-lexical.mdx]: Oh okay I see. TBH I think it's a bit sad to not have this kind of information from a customer standpoint but I understand your point 🙂

Comment (CTNicholas) [reply docs/pages/api-reference/liveblocks-react-lexical.mdx]: Yeah, I hear you! Perhaps if we had a more subtle way to write it down, we could do it.

**Comment by CTNicholas** [comment]
> @CTNicholas did u know why `Tests / e2e-tests ` fails almost always?

We just have some flaky tests that fail often. If the "Merge" button isn't grayed out, you can go ahead and merge. If you can't press the button, re-run the failing tests until you can 👍

**Pull request merged**